src/main/resources/org/jenkinsci/plugins/oic/properties/LogoutQueryParameters/config.jelly

```diff
@@ -0,0 +1,10 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <f:repeatableProperty field="items" header="${%Title}" hasHeader="true">
```

Comment:
I think we should enforce at least one entry here?  if you don't want any entries you would delete the property right?
```suggestion
  <f:repeatableProperty field="items" header="${%Title}" hasHeader="true" minimum="1">
```